.babelrc

```diff
@@ -10,7 +10,8 @@
         [
           "@babel/preset-env",
           {
-            "modules": false
+            "modules": false,
+            "loose": true
```

Comment:
yeah if you want this to be smaller it's a good idea. (I'm a maintainer on Babel)
or upgrade to v7 which might have more savings

Reply:
THX, seems we are already using babel V7.

I'm not sure if it's a good approach turning loose mode on. But actually I've got what I want, a little bit small bundle size and no Object.defineProperty compatibility problem in IE8